pulp_2to3_migration/app/serializers.py

```diff
@@ -43,7 +43,10 @@ def validate(self, data):
         """
         schema = json.loads(SCHEMA)
         validator = Draft7Validator(schema)
-        loaded_plan = json.loads(data['plan'])
+        if isinstance(data['plan'], str):
+            loaded_plan = json.loads(data['plan'])
+        elif isinstance(data['plan'], dict):
```

Comment:
Does it make sense to cover the `else`?
Just in case someone provided `list` by mistake, which is still a valid json. What do you think?